kolena/fr/test_run.py

```diff
@@ -75,6 +77,7 @@ class Data:
         model: Model.Data
         test_suites: List[TestSuite.Data]
 
+    @with_event(event_name=EventAPI.Event.EXECUTE_TEST_RUN)
```

Comment:
same comment here, just in case.

Reply:
Yup I've updated this in a [new commit](https://github.com/kolenaIO/kolena/pull/299/commits/7785a316a5453fd96aaf34fd356d9f7134d7bb19#diff-702b5740a8de89705989ecd08651cfb88d01c425ad9308050308f3186eb4b1a2) too. Thanks for double checking!